nexus/tests/output/nexus_tags.txt

```diff
@@ -22,48 +24,48 @@ spoof_login                              /login
-project_instances_delete_instance        /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}
-project_instances_get                    /organizations/{organization_name}/projects/{project_name}/instances
-project_instances_get_instance           /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}
-project_instances_instance_reboot        /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/reboot
-project_instances_instance_serial_get    /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/serial
-project_instances_instance_start         /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/start
-project_instances_instance_stop          /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/stop
-project_instances_migrate_instance       /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/migrate
-project_instances_post                   /organizations/{organization_name}/projects/{project_name}/instances
+instance_console                         /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/serial
+instance_create                          /organizations/{organization_name}/projects/{project_name}/instances
+instance_delete                          /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}
+instance_disk_attach                     /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/disks/attach
+instance_disk_detach                     /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/disks/detach
+instance_disk_list                       /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/disks
+instance_list                            /organizations/{organization_name}/projects/{project_name}/instances
+instance_migrate                         /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/migrate
+instance_nic_create                      /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/network-interfaces
+instance_nic_delete                      /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/network-interfaces/{interface_name}
+instance_nic_list                        /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/network-interfaces
+instance_nic_update                      /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/network-interfaces/{interface_name}
+instance_nic_view                        /organizations/{organization_name}/projects/{project_name}/instances/{instance_name}/network-interfaces/{interface_name}
```

Comment:
Even if it's very long, I think I might go for `instance_network_interfaces_*` instead of `instance_nic_*`. I like the idea of clarity, and the CLI has bash completion so it shouldn't be too much of a pain to type? What do you think?

Reply:
ok cool; I'll make that change